Gemfile

```diff
@@ -11,72 +11,55 @@ end
 
 gem 'turbolinks'
 
+gem 'pry-rails', group: :development
```

Comment:
We already have `development` group in the Gemfile. Please move this gem into it.